<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cereals</class>
 <widget class="QDialog" name="cereals">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крупы</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>80</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Рис</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Макароны</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Горох</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Крупа гречневая</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Крупа манная</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Пшено</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Нут</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Крупа перловая</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>350</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Сахар</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Соль</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
